package.json

```diff
@@ -7,7 +7,10 @@
   "scripts": {
     "build": "echo FIXME, taskcluster requires a build target."
   },
-  "devDependencies": {},
+  "devDependencies": {
```

Comment:
I wonder if we should figure out how we run `tsc` on these files before publishing to NPM, and include the build command in this PR. That way study authors can use either the ts or js-compiled version. Would be happy to wait if the goal is to just get typescript in this repo!

Reply:
I think that's a good idea, I ran `tsc` manually to make sure it worked but having a basic build command would be easy enough to do now.